### PR TITLE
New product-tour promo link style

### PR DESCRIPTION
**Pull request by keirog**

### CODE REVIEW

**Comment by phamann** [comment]
Can haz screenshot?

**Comment by keirog** [comment]
Ha, hang on am just attaching!

**Comment by keirog** [comment]
**"Default" grid layout**

Before|After
---|---
before-m | after-m

**"S" grid layouts and up**

Before|After
---|---
before | after

**Comment by laurieboyes** [comment]
:+1: :+1: :+1:

File: views/partials/nav.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 			<h1 class="sub-header__title" data-o-grid-colspan="12 S8 XL9">
 				<i class="sub-header__profile-icon"></i><span class="sub-header__myft-icon">My FT</span>
+				{{#if @root.flags.myftProductTourPromo}}<a href="/myft/product-tour" class="myft__tour-link" data-trackable="product-tour-promo">(What’s this?)</a>{{/if}}
```
Comment (phamann) [inline views/partials/nav.html]: What about a `title=` attribute with `Take a tour`?

Comment (keirog) [inline views/partials/nav.html]: Ta, added here https://github.com/Financial-Times/next-myft-page/pull/571 (I went for something longer to give a bit more context)

**Pull request merged**